platformio.ini

```diff
@@ -499,6 +499,25 @@ platform = ${common.platform_wled_default}
 board_build.ldscript = ${common.ldscript_2m512k}
 build_flags = ${common.build_flags_esp8266} -D LEDPIN=3 -D BTNPIN=1
 
+[env:esp32_m5stack_atom_lite]
+board = pico32
+platform = espressif32@2.0
+platform_packages = ${esp32.platform_packages}
+build_unflags = ${common.build_unflags}
+build_flags = ${common.build_flags_esp32} -D WLED_DISABLE_ALEXA -D WLED_DISABLE_BLYNK -D WLED_DISABLE_CRONIXIE
+  -D WLED_DISABLE_HUESYNC -D WLED_DISABLE_INFRARED -D WLED_DISABLE_MQTT
```

Comment:
Random WLED user here: 

I understand why you are disabling Alexa, Hue, MQTT, etc (I did the same with services I don't use), but it shouldn't be hardware specific and it shouldn't be in master branch.

Reply:
Good point, I must have copied my config from an ESP8266 config that had those things disabled.  I'm just going to delete this PR anyway, as the important part (I2S pin redefinition support) I already committed to the dev branch